actions/build-and-push-helm/action.yaml

```
@@ -51,7 +54,9 @@ runs:
 
     - name: Publish chart to ACR
       run: |
-        helm push ${{ steps.set-chart-name.outputs.chart }}-${{ steps.set-chart-version.outputs.ver }}.tgz \
-          oci://${{ inputs.registry }}/helm/${{ inputs.domain }}
+        CHART="${{ steps.set_chart_name.outputs.chart }}"
+        VER="${{ steps.set_chart_version.outputs.ver }}"
+        FILE="$CHART-$VER.tgz"
+        helm push "$FILE" oci://${{ inputs.registry }}/helm/${{ inputs.domain }}
```

You can just fix the bug, no need to set all these variables
```suggestion
        helm push ${{ steps.set_chart_name.outputs.chart }}-${{ steps.set_chart_version.outputs.ver }}.tgz \
          oci://${{ inputs.registry }}/helm/${{ inputs.domain }}
```